docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md

```diff
@@ -0,0 +1,94 @@
+---
+canonical: https://grafana.com/docs/grafana/latest/alerting/configure-notifications/manage-contact-points/integrations/configure-email/
+description: Configure the Email integration to connect alerts generated by Grafana Alerting
```

Comment:
Configure email integration to send email notifications when your alerts are firing